src/domain/home/form/partial/birthdateFormField.module.scss

```diff
@@ -23,6 +23,13 @@
     display: grid;
     grid-template-columns: 1fr 0.2fr 1fr 0.2fr 1fr;
     align-items: baseline;
+
+    /* Hide number input spinners */
```

Comment:
This is unrelated, and was added by accident. It is desirable though.

Reply:
I think you are still able to see the spinners in FF. Noticed by accident.